src/Presence.ts

```diff
@@ -98,7 +98,7 @@ class Presence {
         }
 
         try {
-            await MatrixClientPeg.get().setPresence(this.state);
+            await MatrixClientPeg.get().setPresence({presence: this.state});
```

Comment:
Wow, an actual bug fix?

Reply:
apaprently we migrated to an object long ago and maintained this signature, but the types don't like it so I fixed it :P 

I suspect it's net zero for bugs, unfortunately.